readthedocs/builds/tasks.py

```diff
@@ -187,7 +187,7 @@ def run(self, *args, **kwargs):
 
         lock_id = '{0}-lock'.format(self.name)
         days = kwargs.get('days', 14)
-        limit = kwargs.get('limit', 5000)
+        limit = kwargs.get('limit', 2000)
```

Comment:
If we are worried about the query being killed, why we don't use a lower number here and trigger it more frequently instead?

Reply:
The task is being executed every 30  minutes currently, I thought that was okay, but yeah, we can do something like 1500 builds each 15 min.

Reply:
As long as we are archiving builds faster than producing them, it's fine.